Package.swift

```diff
@@ -0,0 +1,29 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "immer",
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "immer",
+            targets: ["immer"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
+        .target(
+            name: "immer",
+            dependencies: [],
+            path: ".",
+            sources: ["spm.cpp"],
+            publicHeadersPath: ".")
+    ],
+    cxxLanguageStandard: .cxx20
```

Comment:
Immer is actually C++14 compatible, would it be possible to lower this?

Reply:
Yep!